providers/defltprov.c

```diff
@@ -371,6 +371,8 @@ static const OSSL_ALGORITHM deflt_kdfs[] = {
 static const OSSL_ALGORITHM deflt_keyexch[] = {
 #ifndef OPENSSL_NO_DH
     { "DH:dhKeyAgreement", "default=yes", dh_keyexch_functions },
+    { "X25519", "default=yes", x25519_keyexch_functions },
```

Comment:
Is there a OPENSSL_NO_ECX (or should it use NO_EC?)

Reply:
Should be OPENSSL_NO_EC. Done